### PR TITLE
Ingress removal storage service

### PR DESCRIPTION
**Pull request by IsaiasA1**
# Description

Updated documentation on the Karavictl, and Authorization pages to cover the changes made by this PR: (https://github.com/dell/karavi-authorization/pull/224) which covers storage-service being removed from the Ingress.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/725|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by sharmilarama — DISMISSED** [review]

File: content/docs/authorization/cli.md [hunk 1]
```diff
@@ -615,7 +617,9 @@ karavictl storage [flags]
 ##### Options
 
 ```
-  -h, --help   help for storage
+  --addr string   address of the csm-authorization storage service
```
Comment (shaynafinocchiaro) [inline content/docs/authorization/cli.md:620]: Now that addr is the same for tenant and storage service, lets keep the description consistent to be `Address of the server (default "localhost")`

**Review by alikdell — APPROVED** [review]

File: content/docs/authorization/cli.md [hunk 2]
```diff
@@ -615,7 +617,9 @@ karavictl storage [flags]
...
+  -h, --help      help for storage
+  --insecure      insecure skip verify
```
Comment (shaynafinocchiaro) [inline content/docs/authorization/cli.md:622]: If this insecure flag serves the same purpose as the insecure flag for storage, lets keep the description consistent -  `Skip certificate validation`

File: content/docs/authorization/cli.md (hunk 1 above)
Comment (IsaiasA1) [reply content/docs/authorization/cli.md]: fixed

**Review by mgandharva — APPROVED** [review]
lgtm

**Review by shaynafinocchiaro — APPROVED** [review]

**Pull request merged**